website/decorators.py

```diff
@@ -51,11 +51,13 @@ def turn_by_turn_view(func):
 	@wraps(func)
 	def wrap(request, game, player, turn=None, *args, **kwargs):
 		if turn is None:
-			turn = game.current_turn - 1
+			turn = game.current_turn
 		turn = int(turn)
 
-		if turn >= game.current_turn:
+		if turn > game.current_turn:
 			raise Http404("This turn has not yet been played.")
+			# We probably shouldn't raise a 404: that's way too extreme.
```

Comment:
404: DoesNotExists.

Plus, the player should have no way to get here, unless he explictily change the URL.